css/style.css

```diff
@@ -1,7 +1,7 @@
 /** global element styling **/
 
 @import 'https://fonts.googleapis.com/css?family=Lato:400,700';
-
+/*Make it Respnsive*/
```

Comment:
- [ ] As I mentioned earlier, kindly consider suggesting changes in a polite way and also describing `why changes are required`. you can say `I would suggest .... because ....` or `Kindly consider making your website ...... because ....`
-  Consider these project requirements while giving the review:
   -  Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it?
   -  Remember to explain why you request a change - it might be obvious for you but not for someone else.
   -  Remember to be polite and clear in your comments - your job here is to help another person improve in a friendly way.